linopy/model.py

```diff
@@ -969,6 +969,7 @@ def solve(
         sanitize_infinities: bool = True,
         slice_size: int = 2_000_000,
         remote: None = None,
+        stdout: bool | None = None,
```

Comment:
```suggestion
        log: bool | None = None,
```
I guess log is not really the right name here, but better to keep it consistent.
Please also add it to the docstring and briefly explain its logic.

Reply:
it is especially confusion as solve already has a `log_fn` to provide a log filename as well. But yes, consistency is important.

What do you think about renaming it to `progress` - as it basically toggles to show a progress bar?